regression/kernel/kmalloc_invalid_allocation/test.desc

```diff
@@ -0,0 +1,3 @@
+CORE
+test.c
+^VERIFICATION FAILED$
```

Comment:
Sorry, one more thing to change: Our testing script (regression/testing_tool.py) expects the first 3 lines in `test.desc` to be
1. tag (CORE, THOROUGH, KNOWNBUG, etc.)
2. file name of the main source
3. other source files and command-line options for ESBMC

Only starting at the 4. line are the regular expressions placed. Just add an empty 3rd line in the `test.desc` files, please.

Reply:
> Sorry, one more thing to change: Our testing script (regression/testing_tool.py) expects the first 3 lines in `test.desc` to be
> 
> 1. tag (CORE, THOROUGH, KNOWNBUG, etc.)
> 2. file name of the main source
> 3. other source files and command-line options for ESBMC
> 
> Only starting at the 4. line are the regular expressions placed. Just add an empty 3rd line in the `test.desc` files, please.

hi，I have fixed the codes you mentioned. And for the path of gfp.h, the /linux path is not missing because the way I define the direcotry is ubuntuxxx/kernelxxx/include, since there would only be one folder in include directory, I use the /include for the directory of my headers indtead. Please let me know if there are further issues

Reply:
I thought you wanted to model the directory structure of the Linux kernel. For instance, my gfp.h is here:
`/usr/src/linux/include/linux/gfp.h`.

Reply:
> I thought you wanted to model the directory structure of the Linux kernel. For instance, my gfp.h is here: `/usr/src/linux/include/linux/gfp.h`.

hi, i have changed that to /include/linux. could you check :) ?